tests/test_psfr.py

```diff
@@ -280,3 +305,9 @@ def test_combine_psf():
     assert diff_input > diff_output
 
     npt.assert_raises(ValueError, combine_psf, kernel_list_input, kernel, stacking_option='BAD')
+
+
+
+
+
+test_fit_centroid_pso()
```

Comment:
you don't need to execute the definition here. py.test does it anyways :)
This is my last request before approving :)